### PR TITLE
Buffs my little cogchamps.

### LINKED ISSUE #3666: Clockcult: Issues with nonhuman cultists
Eminence has no way to tell who is cult or non cult, they don't have the antag HUD like human servants do.

The Eminence can nuzzle mobs and swim in the pool despite being incorporeal.

Same with scarabs and marauders they don't have access to the heirophant network and can't see the HUD.

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

A variety of clockcult buffs, read the changelog.

## Why It's Good For The Game

They are too weak and are being rolled every round, adjusting their balance is required.

## Changelog
:cl:
add: Clockcult weapons will now automatically wield when you pick them up.
balance: Phazons can no longer phase into areas with the no_jaunt flag.
balance: Clockwork walls are no jaunt proof.
balance: The celestial gateway has had its health buffed from 400 to 1000
balance: Clockcult armour has been significantly buffed, previously it was very weak and made when I poorly understood general armour values.
balance: Marauders no longer automatically repair their shield and require a welder to repair
balance: Linked abscond now has a cooldown of 3 minutes
balance: Abscond now takes 2.5 seconds instead of 3.5 seconds to invoke.
balance: Abstraction crystal health buffed from 100 to 200.
balance: Clockwork armaments cost reduced from 250 to 150
balance: Dimensional breach cost reduced from 10000 to 5000
balance: Interdiction lens range increased from 3 to 5
balance: Interdiction lens now EMP mechs rather than use power.
balance: Ocular warden cost reduced from 500 to 400
balance: Prosperity prism active power used decreased from 4 to 2 per tick.
tweak: Sentinel's compromise now plays a sound on use.
balance: Sigil of vitality power cost reduced from 400 to 300
balance: Sigil of vitalities will now turn people into cogscarabs rather than just husking.
balance: Stargazer power cost reduced from 600 to 300
balance: Marauder cost reduced from 8000 to 2000. Vitality cost increased from 80 to 100.
balance: Vanguard no longer stamcrits you after use.
tweak: eminence can break lightbulbs
tweak: Eminence can use APC UI.
/:cl:

### CODE REVIEW

**Comment by Effemu** [comment]
coggers!

**Comment by MNarath1** [comment]
Clockies can use mechs to get trough the servant barrier i guess the barrier does not check the mobs in the content list of the mech.

**Comment by MNarath1** [comment]
you probably can use a locker or anything that has a content list too for that gonna make a proper issue about this

**Comment by Cenrus** [comment]
Miner borg can spam sentinels compromise without consequence, you may want to have it deal small amount of burning damage each time

File: code/modules/antagonists/clock_cult/servant_of_ratvar.dm [hunk 1]
```diff
@@ -59,7 +59,8 @@
 /datum/antagonist/servant_of_ratvar/apply_innate_effects(mob/living/M)
 	. = ..()
 	owner.current.faction |= "ratvar"
-	owner.language_holder.grant_language(/datum/language/ratvar, TRUE, TRUE, LANGUAGE_CULTIST)
+	var/datum/language_holder/language = owner.get_language_holder()
+	language.grant_language(/datum/language/ratvar, TRUE, TRUE, LANGUAGE_CULTIST)
```
Comment (park66665) [inline code/modules/antagonists/clock_cult/servant_of_ratvar.dm]:
```suggestion
	var/datum/language_holder/LH = owner.current.get_language_holder()
	LH.grant_language(/datum/language/ratvar, TRUE, TRUE, LANGUAGE_CULTIST)
```

And the same goes for `remove_innate_effects`.

**Review by park66665 — APPROVED** [review]
~~All clear. This PR is good to go.~~

YOLO

**Pull request merged**